### PR TITLE
Adds T1 TCs for nautilus branch as well as for Jenkins pipeline

### PR DESCRIPTION
**Pull request by udaysk23**
# Description

Adds T1 TCs for nautilus branch as well as for Jenkins pipeline

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: conf/nautilus/rgw/tier_1_rgw_multisite.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        role:
+          - mon
+          - osd
+
+      node5:
+        disk-size: 20
+        no-of-volumes: 4
+        role:
+          - osd
+          - rgw
```
Comment (psathyan) [inline conf/nautilus/rgw/tier_1_rgw_multisite.yaml:38]: Let's avoid co-locating `rgw` with `osd`. This is what we document in our public facing documents.

File: pipeline/4/jenkinsfile-tier-1-object.groovy [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/*
+    Pipeline script for executing Tier 1 object test suites for RH Ceph 5.0.
```
Comment (psathyan) [inline pipeline/4/jenkinsfile-tier-1-object.groovy:2]: The document string points to 5.0. Please change it to 4.x

File: suites/nautilus/rgw/tier_1_rgw_multisite.yaml [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+  - test:
+      name: Buckets Versioning test
+      desc: test_versioning_objects_acls on secondary
+      module: sanity_rgw_multisite.py
+      clusters:
+        ceph-sec:
+          config:
+            script-name: test_versioning_with_objects.py
+            config-file-name: test_versioning_objects_acls.yaml
+            verify-io-on-site: ["ceph-pri"]
```
Comment (psathyan) [inline suites/nautilus/rgw/tier_1_rgw_multisite.yaml:274]: The cluster names are different

File: suites/nautilus/rgw/tier_1_rgw_multisite.yaml [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+      name: create user
+
+  - test:
+      clusters:
+        ceph-sec:
+          config:
+            config-file-name: test_Mbuckets_with_Nobjects.yaml
+            script-name: test_Mbuckets_with_Nobjects.py
+            timeout: 300
+            verify-io-on-site: [ "ceph-pri" ]
```
Comment (psathyan) [inline suites/nautilus/rgw/tier_1_rgw_multisite.yaml:102]: The primary cluster name is different

File: conf/nautilus/rgw/tier_1_rgw_multisite.yaml (hunk 1 above)
Comment (udaysk23) [reply conf/nautilus/rgw/tier_1_rgw_multisite.yaml:38]: Done

File: pipeline/4/jenkinsfile-tier-1-object.groovy (hunk 2 above)
Comment (udaysk23) [reply pipeline/4/jenkinsfile-tier-1-object.groovy]: Done

File: suites/nautilus/rgw/tier_1_rgw_multisite.yaml (hunk 3 above)
Comment (udaysk23) [reply suites/nautilus/rgw/tier_1_rgw_multisite.yaml]: Done

File: suites/nautilus/rgw/tier_1_rgw_multisite.yaml (hunk 4 above)
Comment (udaysk23) [reply suites/nautilus/rgw/tier_1_rgw_multisite.yaml]: Done

**Comment by psathyan** [comment]
Nice work. The PR looks good. Before the approval, could you please post the logs for the suite runs

**Comment by udaysk23** [comment]
Following are the log location for single as well as multi-site

Single site: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1625125705398
Multi site: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1625125691986

There are some failures in multisite suite for "sync got stuck"  phase

**Comment by udaysk23** [comment]
@psathyan As discussed, I have removed MS suite from pipeline

**Review by psathyan — APPROVED** [review]

**Pull request merged**